fecgov/openFEC
# drop unused fec vsum sched tables, correct views with columns of data type unknown, switch MV definition to use base tables

## Summary (required)

- Resolves #[3908](https://github.com/fecgov/openFEC/issues/3908)
drop unused fec vsum sched views, 
correct views with columns of data type unknown, 
For MVs that use these fec vsum sched views, switch MV definition to use fec_fitem_sched tables 

## How to test the changes locally
- download branch to local machine, run flyway migration (or create_sample_db), make sure migration funs successfully.
- run pytest, make sure all pytest pass successfully.
- The following 3 MVs's definition had been updated to use fec_fitem_sched_xx tables and/or base tables for fec_vsum_fxx tables, a _tmp version of MVs had been created in DEV.  
ofec_reports_ie_only_mv_tmp 
ofec_communication_cost_mv_tmp 
ofec_communication_cost_aggregate_candidate_mv_tmp 

After the _tmp version and the current version of the MVs both refreshed, compare content of the data, it should match.

- In the local machine, change the following lines in models file to point to the _tmp MVs.  The API should behave the same as in PRD.

(in webservices/common/models/reports.py)
class CommitteeReportsIEOnly(PdfMixin, BaseModel):
    __tablename__ = 'ofec_reports_ie_only_mv_tmp'

(in webservices/common/models/costs.py)
class CommunicationCost(db.Model):
    __tablename__ = 'ofec_communication_cost_mv_tmp'

(in webservices/common/models/aggregates.py)
class CommunicationCostByCandidate(BaseSpendingAggregate):
    __tablename__ = 'ofec_communication_cost_aggregate_candidate_mv_tmp'

## Impacted areas of the application
List general components of the application that this PR will affect:
- communication cost
/communication-costs​/
/communication_costs/by_candidate/

- financial
​/reports​/{committee_type}​/


## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
